filebeat/module/postgresql/csv/_meta/fields.yml

```
@@ -0,0 +1,71 @@
+- name: log
```

This is going to generate duplicated field groups, this name should be unique in the module.
```suggestion
- name: csv
```